pom.xml

```diff
@@ -334,16 +334,6 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
-    <repository>
```

Comment:
Does this not break builds for other modules?

Reply:
No. This only impact website as we bump the plugin version last time. I got the guidance from Apache Maven folks.